### PR TITLE
Purge taxonomy feed URLs

### PR DESCRIPTION
**Pull request by janvitos**
In reference to #326

This code will purge the taxonomy feed URLs in addition to the normal taxonomy URLs.

### CODE REVIEW

File: src/WordPress/Hooks.php [hunk 1]
```diff
@@ -187,8 +187,10 @@ public function getPostRelatedLinks($postId)
 
             foreach ($terms as $term) {
                 $termLink = get_term_link($term);
-                if (!is_wp_error($termLink)) {
+                $termFeedLink = get_term_feed_link($term->term_id, $term->taxonomy);
```
Comment (sejoker) [inline src/WordPress/Hooks.php:190]: @janvitos will this cover both Category feeds or Tag feeds?

**Review by sejoker — APPROVED** [review]

File: src/WordPress/Hooks.php (hunk 1 above)
Comment (sejoker) [reply src/WordPress/Hooks.php:190]: looking into the docs https://developer.wordpress.org/reference/functions/get_term_feed_link/ it seems cover both.

**Review by sejoker — APPROVED** [review]

File: src/WordPress/Hooks.php (hunk 1 above)
Comment (janvitos) [reply src/WordPress/Hooks.php:190]: > will this cover both Category feeds and Tag feeds?

Yes it will. But I see you already got your answer :)

**Review by xtuc — APPROVED** [review]
Nice thanks!

**Review by manatarms — APPROVED** [review]

**Pull request merged**